### PR TITLE
Cleanup and modernize

### PR DESCRIPTION
**Pull request by skade**

### CODE REVIEW

File: presentations/cargo/slides.adoc [hunk 1]
```diff
@@ -62,14 +62,16 @@ The debug profile is default here.
 
 Test use the debug profile per default.
 
-`cargo test` runs all test, including documentation test and compilation of examples!
+`cargo test --all --examples` runs all test, including documentation test and compilation of examples!
```
Comment (jamesmunns) [inline presentations/cargo/slides.adoc]:
```suggestion
`cargo test --all --examples` runs all tests, including documentation test and compilation of examples!
```

File: presentations/cargo/slides.adoc [hunk 2]
```diff
@@ -62,14 +62,16 @@ The debug profile is default here.
...
 
 == `cargo bench`
 
 `cargo bench` runs all benchmarks.
 
 Benchmarks default to the release profile.
 
+Benchmarks only work on the nightly compiler.
```
Comment (jamesmunns) [inline presentations/cargo/slides.adoc]: Is it worth mentioning `criterion` here for stable benches?

File: presentations/cargo/slides.adoc [hunk 3]
```diff
@@ -96,7 +99,7 @@ For more details, check the http://doc.crates.io/manifest.html[docs]
 
 At the first build (or through `cargo update`), cargo calculates a version tree fulfilling the constraints stated in Cargo.toml. If successful, it will be saved in a Lockfile (Cargo.lock).
 
-It is recommendable to always keep the Lockfile under version control! (for repeatable builds)
+It can be considered to always keep the Lockfile under version control! (for repeatable builds)
```
Comment (jamesmunns) [inline presentations/cargo/slides.adoc]: If you want this statement, I would reword as such:

> For repeatable builds, consider keeping the Lockfile under version control

However, this doesn't totally match what I would call "best practice", which would be:

* For binaries, include lock file (maybe delete directly in some nightly CI testing)
* For libraries, *don't* include a lock file

File: presentations/little-helpers/slides.adoc [hunk 4]
```diff
@@ -80,52 +81,20 @@ fn main() {
...
 ----
 
---
-
-<1> This is a shorthand vector initialization macro.
-<2> Taking a slice borrows the memory region the vector owns. `&v[..]` is syntax for "borrow from beginning to end".
-<3> Pushing on vector requires a mutable reference to it, violating borrow rules.
-
---
+This expect the operation to have worked.
```
Comment (jamesmunns) [inline presentations/little-helpers/slides.adoc]:
```suggestion
This expects the operation to have worked.
```

File: presentations/overview/slides.adoc [hunk 5]
```diff
@@ -78,7 +78,9 @@ Many examples in this course are very small, which is why we will also spend tim
 - These properties are guaranteed at compile time and have no runtime cost!
 - Optimizing compiler based on LLVM
 - Features with runtime cost are explicit and hard to activate "by accident"
-- No reflection (Dracula-Language)
+- No reflection
+- Zero-cost abstractions
+- "Pay what you use": Rust has features with a runtime cost in an explicit and visible way. Features not used don't cost anything.
```
Comment (jamesmunns) [inline presentations/overview/slides.adoc]:
```suggestion
- "Pay what you use": Rust has features with a runtime cost in an explicit and visible way. Unused features do not come with an associated cost.
```

File: presentations/traits/slides.adoc [hunk 6]
```diff
@@ -34,30 +34,33 @@ include::./2.rs[]
 
 == Borrowing and Ownership of `self`
 
-It is not unlike normal, but at the beginning somewhat unfamiliar.
+It is like usual, but at the beginning somewhat unfamiliar.
```
Comment (jamesmunns) [inline presentations/traits/slides.adoc]: usual to what? I'm not sure what this sentence is trying to convey.

File: presentations/cargo/slides.adoc (hunk 2 above)
Comment (skade) [reply presentations/cargo/slides.adoc]: We probably should.

File: presentations/cargo/slides.adoc (hunk 3 above)
Comment (skade) [reply presentations/cargo/slides.adoc]: The problem is that the best practice is debatable. If you don't include the lockfile, you lose all info about what your library was tested with during a test run.

File: presentations/traits/slides.adoc (hunk 6 above)
Comment (skade) [reply presentations/traits/slides.adoc]: Like Borrowing and Ownership of other values, I'll change that.

File: presentations/cargo/slides.adoc (hunk 2 above)
Comment (skade) [reply presentations/cargo/slides.adoc]:
```suggestion
Benchmarks can be written using the `criterion` crate.
```

File: presentations/cargo/slides.adoc (hunk 3 above)
Comment (skade) [reply presentations/cargo/slides.adoc]:
```suggestion
The Lockfile should always be kept under version control for applications, for libraries, it can be checked in to provide a repeatable test environment.
```

Comment (skade) [reply presentations/cargo/slides.adoc]: @jamesmunns ^^ Better?

File: presentations/traits/slides.adoc (hunk 6 above)
Comment (skade) [reply presentations/traits/slides.adoc]:
```suggestion
It is like normal ownership and borrowing, but at the beginning somewhat unfamiliar.
```

Comment (skade) [reply presentations/traits/slides.adoc]: @jamesmunns New suggestion
